.github/workflows/test.yaml

```diff
@@ -38,6 +38,9 @@ jobs:
       - name: Install dependencies
         run: bun install
 
+      - name: Generate migrations
+        run: bun run db:generate
+
```

Comment:
i dont think this should be in ci, it should normally not cause anything but since it might change the files it might cause unexpected results in the tests

Reply:
basically in all tests you should choose what db model you are testing by manually creating these migrations and going through them